lib/gds_api/json_client.rb

```diff
@@ -157,7 +157,7 @@ def with_ssl_options(method_params)
     end
 
     def do_request(method, url, params = nil, additional_headers = {})
-      loggable = { request_uri: url, start_time: Time.now.to_f }
+      loggable = { request_uri: url, start_time: Time.now.to_f, govuk_request_id: GdsApi::GovukHeaders.headers[:govuk_request_id] }.compact
```

Comment:
Should we be including all of the headers in GdsApi::GovukHeaders with the logged item? e.g more like: GdsApi::GovukHeaders.headers.merge({ request_uri: url, start_time: Time.now.to_f })I know for the purposes of your particular task just govuk_request_id is of interest, but I'd thought if an app had gone to the effort of setting these headers it feels like it should be part of the debug output.